ignite/handlers/checkpoint.py

```diff
@@ -466,6 +471,10 @@ def _setup_checkpoint(self) -> Dict[str, Dict[Any, Any]]:
             for k, obj in self.to_save.items():
                 if isinstance(obj, (nn.DataParallel, nn.parallel.DistributedDataParallel)):
                     obj = obj.module
+                elif Version(torch.__version__) >= Version("1.9.0") and isinstance(obj, ZeroRedundancyOptimizer):
```

Comment:
Let's make version check once during the import and use a flag or something else instead of here

Reply:
Is it OK now?